### PR TITLE
feature: Redesign settings executions list

### PR DESCRIPTION
**Pull request by joanagmaia**
# Changes proposed ✍️
- Executions list for webhooks/automations
- Improvements on reset buttons and submit buttons states
- Improvement on files structures
- Make dialog component generic so that it can be reusable
- ### Screenshots (front-end changes only)  
		- ...  
        
## Checklist ✅
- [x] Label appropriately with `type:feature 🚀`, `type:enhancement ✨`, `type:bug 🐞`, or `type:documentation 📜`.
- [ ] Tests are passing.  
- [ ] New backend functionality has been unit-tested.
- [ ] Environment variables have been updated
  - [ ] Front-end: `frontend/.env.dist`
  - [ ] Backend: `backend/.env.dist`, `backend/.env.dist.staging`, `backend/.env.dist.staging`.
  - [ ] [Configuration docs](https://docs.crowd.dev/docs/configuration) have been updated.
  - [ ] Team members only: update environment variables in Password manager and update the team
- [ ] API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).  
- [ ] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.  
- [ ] All changes have been tested in a staging site.  
- [ ] All changes are working locally running crowd.dev's Docker local environment.

### CODE REVIEW

File: frontend/src/modules/automation/components/webhooks/webhook-form.vue [hunk 1]
```diff
@@ -232,15 +234,18 @@ import { i18n } from '@/i18n'
 import integrationsJson from '@/jsons/integrations.json'
 import activityTypesJson from '@/jsons/activity-types.json'
 import UrlField from '@/shared/fields/url-field'
-import _ from 'lodash'
+import isEqual from 'lodash/isEqual'
```
Comment (mariobalca) [inline frontend/src/modules/automation/components/webhooks/webhook-form.vue:237]: Thanks for reminding me about this, I might need to do this change in other places

File: frontend/src/premium/user/components/list/user-list-filter.vue [hunk 2]
```diff
@@ -0,0 +1,227 @@
+<template>
+  <div class="filter">
+    <app-teleport to="#teleport-user-filter-toggle">
+      <app-filter-toggle
+        :active-filters-count="activeFiltersCount"
+        :expanded="expanded"
+        class="mr-1"
```
Comment (mariobalca) [inline frontend/src/premium/user/components/list/user-list-filter.vue:7]: We can delete this file right?

File: frontend/src/premium/user/pages/user-edit-page.vue [hunk 3]
```diff
@@ -136,6 +150,7 @@ export default {
     },
 
     async doSubmit() {
+      console.log(this.model)
```
Comment (mariobalca) [inline frontend/src/premium/user/pages/user-edit-page.vue:153]: Let's remove this console.log please

File: frontend/src/premium/user/components/list/user-list-filter.vue (hunk 2 above)
Comment (joanagmaia) [reply frontend/src/premium/user/components/list/user-list-filter.vue:7]: This file has the logic to fetch for the initial list. But I can try to see if I can either delete it and move that logic to the main component or just remove what's inside it that is not needed

Comment (mariobalca) [reply frontend/src/premium/user/components/list/user-list-filter.vue:7]: Yeah we can just put the `doFetch` on the list-page or something

**Pull request merged**